Sources/SwiftKuery/ConnectionPoolConnection.swift

```diff
@@ -145,6 +179,8 @@ public class ConnectionPoolConnection: Connection {
         connection.execute(raw, parameters: parameters, onCompletion: onCompletion)
     }
     
+    // MARK: Prepared statements
+
```

Comment:
Further down in the execute() function calls please can you change the Parameter definition for 
- `public func execute(_ raw: String, parameters: [String:Any?], onCompletion: @escaping ((QueryResult) -> ()))`
to be `/// - Parameter query: A string that contains the query to execute.` instead of `A String with the query to execute.`

- `public func execute(_ raw: String, onCompletion: @escaping ((QueryResult) -> ()))`
to be `/// - Parameter query: A string that contains the query to execute.` instead of `A String with the query to execute.`

- `public func execute(_ raw: String, parameters: [String:Any?], onCompletion: @escaping ((QueryResult) -> ()))`
to be `/// - Parameter query: A string that contains the query to execute.` instead of `A String with the query to execute.`